convert-sounds.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sounds_path=$(dirname "$0")/data/sounds
+
+find "$sounds_path" -type f -iregex ".+\\.wav" | while read -r input; do
+    output=${input%.wav}.webm
+    ffmpeg -y -i "${input}" -c:a libopus -compression_level:a 10 -b:a 64k -vn -dash 1 "$output" < /dev/null
```

Comment:
`ffmpeg` is not available in the Hack OS. Where is this script supposed to run?

Reply:
I think the script should live inside a `./tools/` directory in the root of the repository, for consistency with other hack git repositories.

Reply:
> I think the script should live inside a ./tools/ directory in the root of the repository, for consistency with other hack git repositories.

I agree

Reply:
> `ffmpeg` is not available in the Hack OS. Where is this script supposed to run?

There are some ffmpeg binaries IIRC. I tried it once in Endless.

Reply:
Yes `ffmpeg` is inside the Flatpak sandbox, not outside. So this tool should be exposed with Flatpak, and documented in the README.

Reply:
Is there a way to run `ffmpeg` directly with the `flatpak` command line tool?

Reply:
Use `flatpak run --command=ffmpeg org.gnome.Sdk ...`

Reply:
I agree, that is why I suggest that this should be used
```
flatpak run --command=ffmpeg org.gnome.Sdk//3.30 -y -i "${input}" -c:a libopus -compression_level:a 10 -b:a 64k -vn -dash 1 "$output" < /dev/null
```

Reply:
Ok, I'll use the @cfoch approach.